templates/Image_Stitching.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<p>I would give a brief explanation here of what this format means (in case it's been a while since they ran toffy). Also, some more guidance that there are three different types of images to stitch together, each has its own notebook cell, etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/702/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>